simulators/ethereum/engine/suites/cancun/steps.go

```diff
@@ -802,13 +814,17 @@ func (step DevP2PClientPeering) Execute(t *CancunTestContext) error {
 }
 
 func (step DevP2PClientPeering) Description() string {
-	return fmt.Sprintf("DevP2PClientPeering: client %d", step.ClientIndex)
+	return fmt.Sprintf("DevP2PClientPeering: client %d, connection maintained: %b ", step.ClientIndex, step.MaintainConnection)
```

Comment:
Ahh will update this for `DevP2PRequestPooledTransactionHash` too

Reply:
Added here: https://github.com/ethereum/hive/pull/1052/commits/5178de4f0d95e988a015e862dd55886e1e7790b5